### PR TITLE
Add some notes about using the nick as the server

### PR DESCRIPTION
**Pull request by lol768**
https://gist.github.com/lol768/4ce82a8029c103ccaa3d05df279e4922

### CODE REVIEW

File: src/main/java/org/kitteh/irc/client/library/command/WhoisCommand.java [hunk 1]
```diff
@@ -48,7 +48,11 @@ public WhoisCommand(@Nonnull Client client) {
 
     /**
      * Sets the server to query.
-     *
+     *
```
Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/WhoisCommand.java]: Space!

Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/WhoisCommand.java]: I'd probably put a `<p>` there, actually.

File: src/main/java/org/kitteh/irc/client/library/command/WhoisCommand.java [hunk 2]
```diff
@@ -48,7 +48,11 @@ public WhoisCommand(@Nonnull Client client) {
 
     /**
      * Sets the server to query.
-     *
+     * 
+     * It's commonly possible to specify the user's nick here instead of a server,
+     * which will cause the IRC daemon to automatically send it to the user's
+     * current server.
+     *
```
Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/WhoisCommand.java]: Space!

**Comment by lol768** [comment]
I have amended the commit.

**Comment by lol768** [comment]
Thanks!

**Comment by mbax** [comment]
You're welcome! :D

**Pull request merged**